spec/rubocop/ast/regexp_node_spec.rb

```diff
@@ -140,4 +140,24 @@
       it { expect(content).to eq("\n.+\n") }
     end
   end
+
+  describe '#has_interpolation?' do
```

Comment:
It's actually `interpolation?`...

Reply:
I renamed the method but forgot here. Changed now.